SECURITY.md

```diff
@@ -31,7 +31,8 @@ For example, Mautic 3.1 will continue receiving security advisories until the re
 |3.1|17 Aug 2020|24 Aug 2020|23 Nov 2020|30 Nov 2020
 |3.2|23 Nov 2020|30 Nov 2020|16 Feb 2021|22 Feb 2021
 |3.3|16 Feb 2021|22 Feb 2021|17 May 2021|24 May 2021
-|4.0|17 May 2021|TBC 2021|TBC 2022|TBC 2022
+|4.x|16 Feb 2021|30 Aug 2021|26 Sept 2022|27 Feb 2023
+|4.0|17 May 2021|30 Aug 2021|29 Nov 2022|29 Nov 2022
```

Comment:
Is it correct that active support for 4.0 (29 Nov 2022) is longer than 4.x (26 Sept 2022)? Shouldn't it be the other way round, just like is it for 3.0 and 3.x? 😊

Reply:
Doh. That should be 2021. Good catch!